test/SlowTests/Core/Commands/Documents.cs

```diff
@@ -115,6 +115,38 @@ public async Task CanPutGetUpdateAndDeleteDocument()
             }
         }
 
+        [Fact]
+        public async Task CanSkipBeyondCountForEmbeddedIterator()
+        {
+            using (var store = GetDocumentStore())
+            {
+                using (var commands = store.Commands())
+                {
+                    var putResult = await commands.PutAsync(
+                        "companies/1",
+                        null,
+                        new Company
+                        {
+                            Name = "testname",
+                            Phone = 1,
+                            Contacts = new List<Contact> { new Contact { }, new Contact { } },
+                            Address1 = "To be removed.",
+                            Address2 = "Address2"
+                        },
+                        new Dictionary<string, object>
+                        {
+                            {"SomeMetadataKey", "SomeMetadataValue"}
+                        });
+
+                    await commands.PutAsync("users/2", null, new User { Name = "testname2" }, null);
+                    
+                    var documents = await commands.GetAsync(10, 25);
```

Comment:
This says (basically `offset 10 limit 25`, no?

Reply:
yes